_data/banking.yml

```
@@ -127,6 +127,7 @@ websites:
       - sms
       - phone
       - email
+    doc: https://www.barclaycardus.com/servicing/security-center?howWeProtectYou=
```

```suggestion
    doc: https://www.barclaycardus.com/servicing/security-center?howWeProtectYou=#loginProcess
```
Even though `howWeProtectYou` and `howWeProtectYou=` both lead to the same page, I'd go for adding the first subPage `loginProcess` where SafeCode™ is explained.